FBSnapshotTestCase/FBSnapshotTestController.h

```diff
@@ -41,6 +51,11 @@ extern NSString *const FBReferenceImageFilePathKey;
 @property (readwrite, nonatomic, assign) BOOL recordMode;
 
 /**
+ Save reference/captured/diffed images to disk after failures. Defaults to YES.
```

Comment:
I'm referring to the images that get saved out in `saveFailedReferenceImage:...`, which saves out the reference image, the image we captured for the test, and the image showing the difference between the two.